web/cypress/integration/pluginTemplate/create-plugin-template-with-route.spec.js

```diff
@@ -41,6 +41,7 @@ context('Create PluginTemplate Binding To Route', () => {
     cy.contains('Next').click();
     cy.get(this.domSelector.nodes_0_host).type(this.data.ip1);
     cy.contains('Next').click();
+    cy.contains('Custom').should('be.visible');
```

Comment:
@juzhiyuan @LiteSun You should know this.

Reply:
we should resolve it in another PR..

Reply:
Let me create a new PR to fix it. It really shouldn't be mixed up.